docs/content/docs/install/manual_demo/_index.md

```diff
@@ -20,9 +20,29 @@ The OSM Manual Install Demo Guide is a step by step set of instructions to quick
   - `git clone https://github.com/openservicemesh/osm.git`
   - `cd osm`
 
-## Build or Download the OSM CLI
 
-Use the [installation guide](../../install) to install the `osm` cli.
+## Download and install the OSM command-line tool
+
+The `osm` command-line tool contains everything needed to install and configure Open Service Mesh.
+The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
+
+Download the 64-bit GNU/Linux or macOS binary of OSM v0.8.2:
+```bash
+system=$(uname -s)
+release=v0.8.2
+curl -L https://github.com/openservicemesh/osm/releases/download/${release}/osm-${release}-${system}-amd64.tar.gz | tar -vxzf -
+./${system}-amd64/osm version
+```
+
+Download the Windows OSM v0.8.2 binary via Powershell:
+```powershell
+wget  https://github.com/openservicemesh/osm/releases/download/v0.8.2/osm-v0.8.2-windows-amd64.zip -o osm.zip
+unzip osm.zip
+.\windows-amd64\osm.exe version
+```
```

Comment:
This needs to go into OS specific sections, else it seems as though both need to be downloaded regardless. Could be a bit confusing if one doesn't have context.

I would reword this to something along the lines of:
```
For Linux and Mac .... :
...

For Windows ... :
...
```

Reply:
Thank you!
It does indeed read like you need both!
Let me take your suggestion and add small subtitles.